### PR TITLE
Update 2DRendererData_overview.md

### PR DESCRIPTION
**Pull request by unity-cchu**
### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
---
### Purpose of this PR
This adds additional documentation about how to use the _CameraSortingLayerTexture shader variable

### CODE REVIEW

File: com.unity.render-pipelines.universal/Documentation~/2DRendererData_overview.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This option is enabled by default. Clear this option to disable the Depth/[Stenc
 
 ## Camera Sorting Layer Texture
 
-The __2D Renderer Data__ specifies how Unity supplies the shader variable `CameraSortingLayerTexture` for use in custom shaders. You should use this data in the same frame, and on the following layers.
+The __2D Renderer Data__ specifies how Unity supplies the shader variable `CameraSortingLayerTexture` for use in custom shaders. You must use this data in the same frame, and on the following layers. Using CameraSortingLayerTexture before it has been capture may result in unexpected results.
```
Comment (Sam-Unity) [inline com.unity.render-pipelines.universal/Documentation~/2DRendererData_overview.md:27]: It is recommended that you use this data in the same frame and on the following layers, as using `CameraSortingLayerTexture` before it has been captured may result in unexpected results.

**Review by Sam-Unity — CHANGES_REQUESTED** [review]
Suggest combining previous sentence with new line.

**Review by phi-lira — APPROVED** [review]

**Review by Sam-Unity — APPROVED** [review]
Approving previous edits and final edit.

**Comment by unity-cchu** [comment]
Backport 2021.2: https://github.com/Unity-Technologies/Graphics/pull/6887

**Pull request merged**